invenio_theme_tugraz/assets/semantic-ui/less/invenio_theme_tugraz/overrides.less

```
@@ -47,5 +54,5 @@ To #000000
 forcing the font changes
 */
 body {
-  font-family: "Source Sans Pro";
+  font-family: Source Sans Pro !important;
```

```suggestion
  font-family: "Source Sans Pro";
```
This change is not necessary anymore, as I mentioned in the call it took more than a body tag to fix it see #69.
I want you to revert all the changes from this file to resolve the conflict.